concourse/scripts/entry.sh

```diff
@@ -161,13 +161,12 @@ setup_gpadmin_bashrc
 # Do the special setup with root permission for the each task, then run the real task script with
 # gpadmin. bashrc won't be read by 'su', it needs to be sourced explicitly.
 case "$1" in
-    test)
-
+    test_and_build)
         # To make fly debug easier
         su gpadmin -c \
             "source /home/gpadmin/.bashrc &&\
-            /home/gpadmin/greenplumpython_src/concourse/scripts/test.sh"
-        ;;
+            /home/gpadmin/greenplumpython_src/concourse/scripts/test_and_build.sh"
+        ;;   
```

Comment:
spaces at the end should be removed.

Reply:
> spaces at the end should be removed.

Just as thorough as you were back in the days. :wink:

Reply:
Yes, I am a heartless tail space-finding machine.